### PR TITLE
New package: fonts-droid-ttf-20150613

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: srcpkgs/fonts-droid-ttf/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+do_install() {
+	vmkdir usr/share/fonts/TTF
+	vinstall DroidSansFallback.ttf 644 usr/share/fonts/TTF
+	vinstall DroidSansFallbackFull.ttf 644 usr/share/fonts/TTF
+	vinstall DroidSansMono.ttf 644 usr/share/fonts/TTF
+}
+
+# if this package needs to be updated, check the below log for any meaningful changes to the droid font ttf's
+# https://android.googlesource.com/platform/frameworks/base/+log/master/data/fonts
```
Comment (maxice8) [inline srcpkgs/fonts-droid-ttf/template]: you can use the changelog= variable, put it below homepage=

File: srcpkgs/fonts-droid-ttf/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+create_wrksrc=yes
+depends="font-util"
+font_dirs="/usr/share/fonts/TTF"
+short_desc="General-purpose Android fonts (CJK support)"
+maintainer="travankor <travankor@tuta.io>"
+license="Apache-2.0"
+homepage="http://www.droidfonts.com/droidfonts/"
+changelog="https://android.googlesource.com/platform/frameworks/base/+log/master/data/fonts"
+distfiles="https://android.googlesource.com/platform/frameworks/base/+archive/${_githash}/data/fonts.tar.gz"
+checksum=@722c954d839cc131c17161956c5117624ceeea3f90e01d4d8ad3d38f374000cc
```
Comment (cr6git) [inline srcpkgs/fonts-droid-ttf/template]: `checksum=@` ??

Comment (maxice8) [reply srcpkgs/fonts-droid-ttf/template]: checksum=@ is used when the code is generated on the fly when downloaded so it changes everytime we fetch it, prefixing @ forces it to check the hash of the content.

File: srcpkgs/fonts-droid-ttf/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+do_install() {
+	vmkdir usr/share/fonts/TTF
+	vinstall DroidSansFallback.ttf 644 usr/share/fonts/TTF
+	vinstall DroidSansFallbackFull.ttf 644 usr/share/fonts/TTF
+	vinstall DroidSansMono.ttf 644 usr/share/fonts/TTF
```
[the remark this answers is not in the document]
Comment (travankor) [reply srcpkgs/fonts-droid-ttf/template]: @cr6git I added the `conflicts`. I am not really happy with `google-fonts-ttf` though (should be a meta-package that points to packages like this). See https://github.com/voidlinux/void-packages/pull/13150

File: srcpkgs/fonts-droid-ttf/template (hunk 2 above)
Comment (cr6git) [reply srcpkgs/fonts-droid-ttf/template]: @maxice8 I didn't know that, thank you!

File: srcpkgs/fonts-droid-ttf/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
```
Comment (cr6git) [inline srcpkgs/fonts-droid-ttf/template]: "No newline at end of file"

**Pull request merged**